bots/mainbot/utilities.js

```diff
@@ -17,7 +25,7 @@ utilities.isOpen = (self, location) => {
 
 // Prepend the round number before logging. May extend to log more information
 utilities.log = (self, message) => {
-	self.log(`Round ${self.me.turn} - ${message}`);
+	self.log(`Round ${self.me.turn} - ${LABELS[self.me.unit]} - ${message}`);
```

Comment:
Good idea. easy way to see in logs what type of bot is logging

Reply:
As long as you're not looking for errors produced by the engine, if you're debugging a particular unit you can just pipe the output through a grep command and easily only see the output for the unit type you're interested in.